<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textFunc1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitulo_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Método de Gauss Chebyshev</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelY_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>f(x) =</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditN1">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelY_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>n =</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textResult1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textResult2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelY_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Resultado Chebyshev</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelY_9">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Resultado Scipy.Integrate</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelY_10">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>450</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Gráfico</string>
    </property>
   </widget>
   <widget class="MplWidget" name="MplWidgetGaussChebyshev" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>490</y>
      <width>451</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCalcular">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelY_11">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>340</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Diferença dos métodos:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditDiferenca">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>340</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>120</y>
      <width>441</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>A quadratura de Gauss Chebyshev é uma 
extensão da quadratura de Gauss.
Para aproximar o cálculo de integrais do tipo:

∫f(x)⁄√(1-x²)dx, no intervalo (-1, 1).

Neste caso, a aproximação é realizada como:

∫f(x)⁄√(1-x²)dx ≈ ∑i(1,n) wᵢf(xᵢ), onde:

xᵢ = cos(((2i -1)π)/2n) e wᵢ = π/n.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sobre</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1122</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
